scripts/go_executable_build.sh

```diff
@@ -37,7 +37,7 @@ if [ "$(uname -s)" == "Darwin" ]; then
    LIB[libmcl.dylib]=${MCL_DIR}/lib/libmcl.dylib
    LIB[libgmp.10.dylib]=/usr/local/opt/gmp/lib/libgmp.10.dylib
    LIB[libgmpxx.4.dylib]=/usr/local/opt/gmp/lib/libgmpxx.4.dylib
-   LIB[libcrypto.1.0.0.dylib]=/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib
+   LIB[libcrypto.1.1.dylib]=/usr/local/opt/openssl/lib/libcrypto.1.1.dylib
```

Comment:
why this bump

Reply:
just because I have this version on my computer.....